src/metabase/pulse/render/table.clj

```diff
@@ -131,8 +141,8 @@
   ([color-selector normalized-zero column-names [header & rows]]
    [:table {:style (style/style {:max-width "100%"
                                  :white-space :nowrap
-                                 :padding-bottom :8px
-                                 :border-collapse :collapse
+                                 :border  (str "1px solid " style/color-border)
+                                 :border-radius :6px
```

Comment:
To be honest, this is where I have to eyeballed the rounded corner, as the spec is 8px round based on 22px font size and that came to 4.36px based on 12px font-size.

I tried 4px and 5px and they still look too small, and 6px seems just right.

#### Design
![image](https://user-images.githubusercontent.com/1937582/190606319-5ed1674e-a8f5-445d-b7e8-0608832a096e.png)

#### Implementation
![image](https://user-images.githubusercontent.com/1937582/190606362-cedb8d75-7a5e-4a01-b2a3-dc7c95449f81.png)